scripts/start

```diff
@@ -0,0 +1,3 @@
+export NODE_ENV=$1
+export PORT=$2
+node --harmony build/server.js
```

Comment:
because we will run `npm start` anyway so it will always the folder in which `package.json` is located